docs/modules/policies/pages/best_practices.adoc

```diff
@@ -263,20 +263,22 @@ Ruby::
 include::example$example.rb[]
 ----
 --
-//Rust::
-//+
-//--
-//[source,rust,linenums]
-//----
-//include::example$example.rs[]
-//----
-//--
+Rust::
++
+--
+[source,rust,linenums]
+----
+include::example$example.rs[]
+----
+--
 ====
 
 You can find a full (and extended) example of the above in our https://play.cerbos.dev/p/IJxlK6131f642ND65F1EhPmiT18Ap1A5[SaaS Workspace Policy playground example].
 
 === Dynamic Policies
 
-There might be circumstances where you want to create brand new resources and actions on the fly. If this is the case, then you can use the xref:api:admin_api.adoc[Admin API] configured alongside a mutable xref:configuration:storage.adoc[database storage engine] to provide this functionality. This would be handled within your application layer, with the desired policy contents provided to the PDP via the API.
+There might be circumstances where you want to create or update resources and actions on the fly; an example of this might be a multi-tenant platform which provides tenants the ability to manage their own policies.
```

Comment:
```suggestion
There might be circumstances where you want to create or update resources and actions on the fly; an example of this might be a multi-tenant platform that provides tenants the ability to manage their own policies.
```